designsafe/apps/api/publications/views.py

```diff
@@ -43,11 +43,16 @@ def get(self, request, operation, project_id, revision=None):
 """
 class PublicationDataCiteView(BaseApiView):
     def get(self, request, doi):
+        url = f'https://api.datacite.org/dois/{doi}' 
```

Comment:
Just checking, is this the preferred solution over adjusting the `get_doi` call?
https://github.com/DesignSafe-CI/portal/blob/1d5d02dbfc30171959ee986ab9e691017624d901/designsafe/apps/projects/managers/datacite.py#L22